### PR TITLE
fix rel_pos_to_abs_pos for t = 1, 2.

### PR DESCRIPTION
**Pull request by zhiyun**

### CODE REVIEW

File: axlearn/common/attention.py [hunk 1]
```diff
@@ -1963,21 +1963,24 @@ def rel_pos_to_abs_pos(x: Tensor) -> Tensor:
...
             absolute position k = i + j - (T - 1), if 0 <= k < T, otherwise the value is not used.
+            T >= 1.
 
     Returns:
         y: a Tensor of shape [T, T], s.t. y[i, k] = x[i, j] where k = i + j - (T - 1),
             if 0 <= k < T.
     """
     t, offset_length = x.shape
     assert offset_length == 2 * t - 1
+    if t == 1:
```
[the remark this answers is not in the document]
Comment (zhiyun) [reply axlearn/common/attention.py]: Done.

File: axlearn/common/attention.py [hunk 2]
```diff
@@ -1963,21 +1963,24 @@ def rel_pos_to_abs_pos(x: Tensor) -> Tensor:
...
     # [t * (2t - 1)].
     x = x.reshape([-1])
     # [t * (2t - 2)].
     x = x[t - 1 : -1]
     # [t, 2t - 2].
     x = x.reshape([t, -1])
-    # [t, t].
-    x = x[:, : -(t - 2)]
+    # [t, t]. When t = 2, do not trim.
+    x = x[:, : -(t - 2) or None]
```
[the remark this answers is not in the document]
Comment (zhiyun) [reply axlearn/common/attention.py]: Done.

**Review by ruomingp — APPROVED** [review]

File: axlearn/common/attention.py [hunk 3]
```diff
@@ -1963,21 +1963,25 @@ def rel_pos_to_abs_pos(x: Tensor) -> Tensor:
     Args:
         x: a Tensor of shape [T, 2*T - 1], where x[i, j] represents the bias between query[i] and
             absolute position k = i + j - (T - 1), if 0 <= k < T, otherwise the value is not used.
+            T >= 1.
```
Comment (markblee) [inline axlearn/common/attention.py:1966]:
```suggestion
            T is expected to be >= 1.
```

**Review by markblee — APPROVED** [review]

File: axlearn/common/attention.py (hunk 3 above)
Comment (zhiyun) [reply axlearn/common/attention.py:1966]: Done.

**Review by markblee — APPROVED** [review]

**Review by markblee — APPROVED** [review]

**Pull request merged**